internal/crq/watch.go

```
@@ -1357,34 +1367,11 @@ func (s *Service) beatDispatch(ctx context.Context, report NextReport, token str
-							if round != nil && round.DispatchHeld(s.clock()) {
-								taken = true
-							}
-							gone = !taken
-							return ErrNoChange
-						}
-						return nil
-					}
-					ok, byOther := round.HeartbeatDispatch(token, s.clock())
-					taken = byOther
-					if !ok {
+					updated := false
```

golangci-lint (2.12.2)

[error] assigned to updated, but reassigned without using the value

(wastedassign)